### PR TITLE
v3.2.0 RC

### PR DESCRIPTION
**Pull request by wachunei**

### CODE REVIEW

File: app/components/UI/Swaps/index.js [hunk 1]
```diff
@@ -378,20 +379,40 @@ function SwapsAmountView({
...
 		},
-		[toggleSourceModal]
+		[toggleSourceModal, setSlippageAfterTokenPress, destinationToken]
 	);
 
 	const handleDestinationTokenPress = useCallback(
 		item => {
 			toggleDestinationModal();
 			setDestinationToken(item);
+			setSlippageAfterTokenPress(sourceToken?.address, item.address);
```
Comment (gantunesr) [inline app/components/UI/Swaps/index.js:413]: Same question here for `sourceToken?.address`

**Review by gantunesr — APPROVED** [review]
LGTM. I just left two question out of curiosity

File: app/components/UI/Swaps/index.js [hunk 2]
```diff
@@ -378,20 +379,40 @@ function SwapsAmountView({
...
+			}
+		},
+		[setSlippage, chainId, isDirectWrapping]
+	);
+
 	const handleSourceTokenPress = useCallback(
 		item => {
 			toggleSourceModal();
 			setSourceToken(item);
+			setSlippageAfterTokenPress(item.address, destinationToken?.address);
```
[the remark this answers is not in the document]
Comment (wachunei) [reply app/components/UI/Swaps/index.js:404]: Nothing, inside `setSlippageAfterTokenPress` `enableDirectWrapping` will be `false` and nothing will happen.

File: app/components/UI/Swaps/index.js (hunk 1 above)
Comment (wachunei) [reply app/components/UI/Swaps/index.js:413]: Same answer =D

**Pull request merged**